config/initializers/launchdarkly.rb

```diff
@@ -1,7 +1,11 @@
 Rails.application.configure do
```

Comment:
I think that the puma config might also be updated, since it creates an LDClient

Reply:
Any idea why that config is needed? :thinking: It seems it can be removed, since `preload_app!` is not used and thus `Rails` is never defined in that block (same for `SemanticLogger`).

Reply:
addressed at https://github.com/rubygems/rubygems.org/pull/3650 for now